framework/Metrics/metrics/StatsTestsUtils.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+
+from scipy import stats
+
+
+def build_segments(array, seg_length: int):
+"""
+This method groups elements of an array into segments of length seg_length
+@In array, array-like containing data to be grouped into segments.
+@In seg_length, size of the segment desired
+@Out, array of segments of length seg_length
+"""
```

Comment:
docstring open/closer should be indented once, and the docstring contents indented twice, as:
```python
def func(args):
  """
    description
    @ In, args, args type, description
    (etc)
  """
  codeLine1 = True
```
then the code should be indented once as well, with 2-space indents everywhere